ui/component/fileActions/view.jsx

```diff
@@ -153,7 +165,7 @@ export default function FileActions(props: Props) {
                 className="button--file-action"
                 icon={ICONS.EDIT}
                 label={isLivestreamClaim ? __('Update or Publish Replay') : __('Edit')}
-                navigate={`/$/${PAGES.UPLOAD}`}
+                navigate={getUri(isLivestreamClaim, isPostClaim)}
```

Comment:
Navigation into `/$/upload` requires 2 _Back_ actions to navigate back.  I think somewhere it is double pushing?

Steps:
1. `/$/uploads` (maybe doesn't need to start from here)
2. Open a Post (maybe doesn't matter what type)
3. Click Edit
4. Click browser's Back 
    - Seem to require 2 clicks, while production only requires one.

Reply:
Nice catch! I had to add the redirects to the `doPrepareEdit` function to make it working in all cases. That made the existing `navigate` redirects redundant. Fixed.

Reply:
One more scenario:
- Enter `/$/uploads`
- Edit a video claim from the context menu
- Need to press back twice.

Reply:
Fixed.